File: js/calculator.js
function addition(a, b) {
    return a + b
}
function substraction(a, b) {
    return a - b
}
function muliplication(a, b) {
    return a * b
}
function division(a, b) {
    try {
        return a / b
    }
    catch (e) {
        console.log(e.message)
    }
}
function modululo(a, b) {
    return a%b
}
a = 5
b = 2
console.log(addition(a, b))
console.log(substraction(a, b))
console.log(muliplication(a, b))
console.log(division(a, b))
console.log(modululo(a, b))

File: c#/Calculator.cs
using System;

namespace ConsoleApp24
{
    class Program
    {
        static int addition(int a, int b)
        {
            return a + b;
        }
        static int substraction(int a, int b)
        {
            return a - b;
        }
        static int multiplication(int a, int b)
        {
            return a * b;
        }
        static double division(int a, int b)
        {
            try
            {
                if (a != 0 || b != 0){
                    return a / b; 
                }
            }
            catch(Exception e)
            {
                Console.WriteLine("An error occured: {0}", e);
            }
            return 0;
        }
        static void Main(string[] args)
        {
            Console.WriteLine(addition(2, 5));
            Console.WriteLine(substraction(2, 5));
            Console.WriteLine(multiplication(2, 5));
            Console.WriteLine(division(1, 1));
        }
    }
}
